symphony/bdk/core/config/model/bdk_certificate_config.py

```diff
@@ -0,0 +1,22 @@
+class BdkCertificateConfig:
+    """Class containing a Certificate configuration
+    """
+    def __init__(self, path=None, content="", password=None, **kwargs):
+        self.path = path
+        self.content = content
+        self.password = password
+
+    def is_configured(self) -> bool:
+        """"Check if the certificate authentication is configured or not
+
+        :return: true if the certificate authentication is configured
+        """
+        return (self.path is not None or self.content != "") and self.password is not None
+
+    def is_valid(self) -> bool:
+        """Check if the certificate configuration is valid.
+        If both certificate path and content, the configuration is invalid.
+
+        :return: true if the RSA configuration is invalid.
+        """
+        return (self.path is not None and self.content != "") and self.password is not None
```

Comment:
I guess it is: `(self.path is not None *or* self.content != "") and self.password is not None`, i.e. you have either a path to a cert file or the cert content.

Reply:
ah I think I misunderstood that one then.
it's probably 
```python
not (self.path is not None and self.content != "") and self.password is not None
```
to enforce only one being used. <br>
In that aspect I suppose, I should do the same for the rsa private key.